rts/Game/Action.h

```diff
@@ -12,13 +12,20 @@ class Action
 	Action() {}
 	Action(const std::string& line);
 
+	typedef std::function<bool (Action, Action)> Comparison;
```

Comment:
Wouldn't these ideally be `const Action &`?
```suggestion
	typedef std::function<bool (const Action &, const Action &)> Comparison;
```

Reply:
I think I tried it and it made using it in `std::` functions not possible, I'll try again.

Reply:
Nevermind, lol. Let me just compile and test, thanks for the contribution!